From aa35f0b91730b844828eec1b538041c1d24d2c95 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Fri, 9 Aug 2019 23:26:07 +0200
Subject: [PATCH 1/1] cmd/bdinfo: sandbox: print the relocation offset

On the sandbox provide the relocation offset. This value can be used for
debugging with GDB using the `add-symbol-file u-boot <reloc off>' command.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 cmd/bdinfo.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cmd/bdinfo.c b/cmd/bdinfo.c
index 86c17dc427..560c039d37 100644
--- a/cmd/bdinfo.c
+++ b/cmd/bdinfo.c
@@ -395,6 +395,7 @@ int do_bdinfo(cmd_tbl_t *cmdtp, int flag, int argc, char * const argv[])
 
 	print_bi_boot_params(bd);
 	print_bi_dram(bd);
+	print_num("reloc off", (uintptr_t)(gd->reloc_off + gd->arch.ram_buf));
 	print_eth_ip_addr();
 
 #if defined(CONFIG_LCD) || defined(CONFIG_VIDEO)
-- 
2.20.1

